core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -461,7 +461,6 @@ impl EthTxManager {
             tx_status.receipt,
             failure_reason
         );
-        panic!("We can't operate after tx fail");
```

Comment:
Concern: with this approach we will be paged with "failed tx" alert only when number_of_inflight_txs==0, it adds latency. I think it makes sense to report metric about failed tx, ideally without waiting for block finalization (this would require some refactoring) and add alert based on it

Reply:
Good idea, I've added a new metric for failed transactions